components/about_build_modal/index.js

```diff
@@ -6,12 +6,9 @@ import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general
 
 import AboutBuildModal from './about_build_modal.jsx';
 
-function mapStateToProps(state, ownProps) {
-    return {
-        ...ownProps,
-        config: getConfig(state),
-        license: getLicense(state)
-    };
-}
+const mapStateToProps = (state) => ({
```

Comment:
I'm also not a huge fan of the implicit return statements like this because of the difference between statements like `() => {value}` vs `() => ({value})`